design/open/pull-request.md

```diff
@@ -0,0 +1,28 @@
+# Pull-request Proposal
+
+Enable a way to submit changes to a branch by creating a request to merge changes.
+The pull-request captures the merge operation we request to apply.
+
+### Goals
+
+- Capture request to merge a branch or specific reference
+- Mechanism to review, discuss and approve changes before applying to a branch
```

Comment:
I don't think `"discuss"` is something that is supported by the below proposal.

Reply:
Added discussion section that will include zero or more comments. Each comment will be added to the end of the discussion.